src/frontend/admin/src/components/templates/courses/form/CourseForm.tsx

```diff
@@ -156,42 +161,52 @@ export function CourseForm({ course, ...props }: Props) {
                   />
                 </Grid>
 
-                <Grid xs={12} md={6}>
-                  <Typography variant="subtitle2">
-                    {intl.formatMessage(
-                      courseFormMessages.productRelationSubtitle,
-                    )}
-                  </Typography>
-                </Grid>
-                <Grid xs={12} md={6}>
-                  <Box display="flex" width="100%" justifyContent="flex-end">
-                    <Button
-                      size="small"
-                      onClick={relationToProductModal.handleOpen}
-                    >
-                      {intl.formatMessage(
-                        courseFormMessages.addRelationButtonLabel,
-                      )}
-                    </Button>
-                  </Box>
-                </Grid>
-                <Grid xs={12}>
-                  <Stack spacing={1}>
-                    {productsArray.fields.map((product, index) => (
-                      <DndDefaultRow
-                        key={`${product.product.title}-${product.id}`}
-                        onDelete={() => productsArray.remove(index)}
-                        mainTitle={product.product.title}
-                        rightActions={
-                          <Button onClick={() => editRelation(product, index)}>
-                            Edit
-                          </Button>
-                        }
-                        subTitle={`${product.organizations.length} organizations selectionnées`}
-                      />
-                    ))}
-                  </Stack>
-                </Grid>
+                {showProductRelationSection && (
+                  <>
+                    <Grid xs={12} md={6}>
+                      <Typography variant="subtitle2">
+                        {intl.formatMessage(
+                          courseFormMessages.productRelationSubtitle,
+                        )}
+                      </Typography>
+                    </Grid>
+                    <Grid xs={12} md={6}>
+                      <Box
+                        display="flex"
```

Comment:
Not a real review comment but just opinion cf the daily of the day: It's the exact same thing as using `<div style=.../>` 😬